### PR TITLE
improve instructions on setting eas.projectId in app.json

### PR DESCRIPTION
**Pull request by dsokal**
Before and after (IMO looks better):
Screenshot 2022-08-25 at 14 24 53

### CODE REVIEW

**Review by wkozyra95 — APPROVED** [review]

File: packages/eas-cli/src/project/projectUtils.ts [hunk 1]
```diff
@@ -96,17 +96,19 @@ export async function saveProjectIdToAppConfigAsync(
     case 'success':
       break;
     case 'warn': {
-      Log.log();
+      Log.warn();
       Log.warn('It looks like you are using a dynamic configuration!');
```
Comment (brentvatne) [inline packages/eas-cli/src/project/projectUtils.ts:100]:
```suggestion
      Log.warn(`Warning: Your project uses dynamic app configuration (${chalk.bold('app.config.js')}), and the EAS project ID can't automatically be added to it.`);
```

replace app.config.js above with the dynamic config file used (app.config.js or ts). this is included in the default `message` (https://github.com/expo/expo/blob/a1b2dcb7558eeb126a4c71322c7df5552923ad59/packages/%40expo/config/src/Config.ts#L407-L410) but it's probably easiest to just get the value again with [`getConfigFilePaths`](https://github.com/expo/expo/blob/a1b2dcb7558eeb126a4c71322c7df5552923ad59/packages/%40expo/config/src/Config.ts#L263-L278)

File: packages/eas-cli/src/project/projectUtils.ts [hunk 2]
```diff
@@ -96,17 +96,19 @@ export async function saveProjectIdToAppConfigAsync(
...
-      Log.log(
+      Log.warn(
         chalk.dim(
           'https://docs.expo.dev/workflow/configuration/#dynamic-configuration-with-appconfigjs)\n'
         )
       );
       Log.warn(
-        'In order to finish setting up your project you are going to need manually add the following to your "extra" key:\n\n'
+        'In order to finish setting up your project you are going to need manually edit app.json/app.config.js and add the following to your "extra" key:'
```
Comment (brentvatne) [inline packages/eas-cli/src/project/projectUtils.ts:107]:
```suggestion
        'To complete the setup process, set the `${chalk.bold('eas.extra.projectId')} in your `${chalk.bold('app.config.js')}`:'
```

replace app.config.js with the actual file name as discussed above

**Review by brentvatne — APPROVED** [review]
thanks for doing this! i added more suggestions because i think it can still be better

**Comment by dsokal** [comment]
/changelog-entry chore Improve instructions on setting `extra.eas.projectId` in app configuration.

**Pull request merged**